### PR TITLE
view comments functionality added

### PR DESCRIPTION
**Pull request by JRDN-13**

### CODE REVIEW

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -18,8 +19,15 @@ function SingleReview() {
     return <p>Loading...</p>;
   }
 
+  function handleClick() {
+    fetchCommentsByReviewId(review_id).then((results) => {
+      console.log(results.comments);
```
Comment (HannahBurks) [inline src/components/SingleReview.jsx]: Remove any stray console logs such as here :)

File: src/components/SingleReview.jsx [hunk 2]
```diff
@@ -28,8 +36,30 @@ function SingleReview() {
...
-      <p>VOTES: {currentReview.votes}</p>
-    </div>
+      <p>
+        by <em>{currentReview.owner}</em>
+      </p>
+      <section className="votes-container">
+        <button>△</button> {currentReview.votes} vote(s)
+        <button>▽</button>
+      </section>
+      <button onClick={handleClick}>Click to view comment(s)</button>
```
Comment (HannahBurks) [inline src/components/SingleReview.jsx:44]: We also want to handle if there are no comments yet on an review, that we display a message such as 'No comments here yet'

File: src/components/ReviewsCard.jsx [hunk 3]
```diff
@@ -7,29 +7,28 @@ function ReviewsCard({ review }) {
...
-          <p>OWNER: {owner}</p>
-          <p>VOTES: {votes}</p>
-          <p>{comment_count} comment(s)</p>
+      <section className="reviews-details">
+        <p>
+          Designed by <em>{designer}</em>
+        </p>
+        <p>category: {category}</p>
+        <section className="votes-container">
+          <button>△</button> {votes} vote(s)
```
Comment (HannahBurks) [inline src/components/ReviewsCard.jsx]: Votes area can be removed for now as it is not relevant to this specific ticket

**Review by HannahBurks — CHANGES_REQUESTED** [review]
A strong start Andrew, just a few tweaks to make. Once amended please send another PR 😃

**Review by dappernerddesigns — APPROVED** [review]
Looks good after those changes. Some things to consider, you could handle your review and comment api calls as a promise all so that you have all the necessary data. That way the loading message is applicable to both. Currently on a slow connection I get a lovely message that its loading, but showing the comments lags and then shows. 

While this is not a problem, it could be tidied up with that method.

A message to the user about not comments is great, but does it need to be held in state? Could you use conditional rendering to show a message or a list of comments based on the comments length?

Looking great so far, just some points above to consider. They are not a 'must have' more of a suggestion of a way to tighten this component up :)

**Pull request merged**